source/components/ChooseHouseSection.js

```diff
@@ -38,7 +38,7 @@ class ChooseHouseSection extends HTMLElement {
         .option {
           order-radius: 0.5rem;
           padding: 1.5rem;
-          max-width: 12rem;
+          max-width: 16rem;
```

Comment:
did you test this? will this overflow on tiny screens?

Reply:
By checking the mobile emulation in chrome. It fits commonly used mobile devices.